### ISSUE TITLE
Permission othersBalance was never checked

### ISSUE THREAD

**Issue by sinux-l5d** [WeiiswurstDev/EasyConomy#2]
Hello,
I'm running a server for some friends and I noticed that even if I didn't give them the permission, they could get the account balance of others. 
Here is a piece of my config.yml : 
```
enable-logging: false

permissions:
  README: >
    IF YOU WANT THE PLUGIN TO USE PERMISSIONS,
    WRITE THE PERMISSIONS DOWN IN THE "".
    EMPTY QUOTATION MARKS => BALANCE AND OTHERSBALANCE AND PAY FOR EVERYONE
    AND MODIFY FOR OP ONLY
  balance: "ec.balance.self"
  othersBalance: "ec.balance.other"
  pay: "ec.pay"
  modify: "ec.modify"
```
And a piece of my bPermission config : 
```
default: default
groups:
  default:
    permissions:
    - ec.balance.self
    - ^ec.balance.other
    - ec.pay
```

I noticed that in `src/main/java/dev/wwst/easyconomy/commands/BalanceCommand.java`, you check if the player has permissions.balance but never permissions.othersBalance.

I would have liked to fix it myself, but I'm still learning Java and I don't know where to find the librairies to compile your code.

I'm right here if you need further informations :)

**Comment by ArvedEh** [comment]
Hi!

I will take a look at that when I finally find some time to do so, although that could take a few days.

If you want to edit the source code, the github repository is actually just an IntelliJ project. This means that you _should_ be able to import it in the free community edition.

I hope that I can look at it soon!

**Comment by ArvedEh** [comment]
fixed with [Release 1.2.1](https://github.com/WeiiswurstDev/EasyConomy/releases/tag/1.2.1)

Thanks for reporting it :heart:

**Comment by sinux-l5d** [comment]
Thank you for fixing it 😘
